From e423b662f45bf3be22e5f4b2e4602d01bbc45a66 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 11 Jun 2023 19:39:57 +0200
Subject: [PATCH 74/84] arm64: dts: rockchip: rk3399-pinephone-pro: Add I2C
 supply to stk3311

It is needed to communicate with the device, when other users of
vcc3v0_touch shut down the regulator.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
(cherry picked from commit 658fe4ad25378e4e8c030623e7bbe11eed74498f)
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 564cbe307d0c..2a2568caf975 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -848,6 +848,7 @@ light-sensor@48 {
 		proximity-near-level = <300>;
 		vdd-supply = <&vcc_3v0>;
 		leda-supply = <&vcc_3v0>;
+		i2c-supply = <&vcc3v0_touch>;
 	};
 };
 
-- 
2.55.0

